packages/ember-htmlbars/tests/integration/attrs_lookup_test.js

```diff
@@ -1,154 +0,0 @@
-import compile from 'ember-template-compiler/system/compile';
```

Comment:
@Joelkang I don't fully understand your question: what was failing without this?